Update CONTRIBUTING.md

martindsq opened the pull request:
Just some ideas to start documenting our processes. I will create a list with items we can document.

mplorentz commented on CONTRIBUTING.md:
````diff
@@ -1,6 +1,65 @@
+```mermaid
+graph TD
+  Planetary((Planetary))
+  Planetary-->Logger
````
Looking at the way the mermaid diagram is rendered, I think it would make more sense to me if the arrows were reversed? Maybe because I think of the imports as a tree with Planetary at the root, but this makes it seem like Logger is the root node. Does your brain see it the opposite way?

martindsq replied:
The way I read A->B is A uses (or depends on) B, just like [the UML dependency relationship](https://www.uml-diagrams.org/dependency.html)

mplorentz replied:
Ok that makes sense.

mplorentz commented on CONTRIBUTING.md:
```diff
@@ -1,6 +1,65 @@
+
+This prevent you from commiting real values in the repository.
+
+## Bug Reports
```
Maybe it would make sense to move the Bug Reports, Translations, and Pull Request sections up a little higher in the file, because I think it will be relevant to more people than the Secrets or Swift Package Creation.

mplorentz commented:
bump

martindsq commented:
I moved the Bug Reports, Translations, and Pull Request sections up, added the steps we have in README.md to set up the development environment and added a few more steps (install swiftlint, for instance), and added the CONTRIBUTING.md file to the Planetary project to see it there alongside the other markdown files

mplorentz commented on CONTRIBUTING.md:
````diff
@@ -1,6 +1,74 @@
+2. Install `rbenv` and `swiftlint` using [Homebrew](https://brew.sh/) and add it to your shell: 
+
+```
+$ brew install rbenv && rbenv init
````
```suggestion
$ brew install rbenv swiftlint && rbenv init
```

mplorentz commented on CONTRIBUTING.md:
````diff
@@ -1,6 +1,74 @@
+$ pod install
+```
+
+6. Install `swiftlint` using [Homebrew](https://brew.sh/) 
````
It looks like this is included in step 2, so this step could be removed.

mplorentz reviewed (approved):
It looks like our swiftlint install instructions could use some small edits, but feel free to merge after that. Thanks for putting some time into this!
